### PR TITLE
Phase2 part2

### PR DESCRIPTION
**Pull request by keitwb**

### CODE REVIEW

**Comment by keitwb** [comment]
Sorry this is so big Jay but I want to at least get somebody to review it.

File: glide.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package: github.com/signalfx/cloudfoundry-bridge
```
Comment (jrcamp) [inline glide.yaml]: Why'd you decide to switch to glide just out of curiosity?

File: metrics/firehose_nozzle.go [hunk 2]
```diff
@@ -125,10 +149,21 @@ func (o *SignalFxFirehoseNozzle) datapointsFromEnvelope(envelope *events.Envelop
...
+		// "index" in the firehose is a long guid value, whereas in bosh hm
+		// metrics, it is a simple cardinal # indicating the instance index
+		// (e.g. 0, 1, 2, etc.).  Call the firehose "index" the same as the
+		// BOSH HM "id" field for consistency.  They appear to be the same
+		// thing.
+        "id": envelope.GetIndex(),
         "metric_source": "cloudfoundry",
     }
 
+	if envelope.GetTags() != nil {
```
Comment (jrcamp) [inline metrics/firehose_nozzle.go]: I don't think you need the nil check as range should be a no-op on nil.

File: glide.yaml (hunk 1 above)
Comment (keitwb) [inline glide.yaml]: To get versioned dependencies.  From what I understand, just using the built-in golang tools always pulls the latest version of a package from the repo.  Glide also handles versioning transitive deps.

File: metrics/firehose_nozzle.go (hunk 2 above)
Comment (keitwb) [inline metrics/firehose_nozzle.go]: It does.  I removed nil check.

**Pull request merged**